draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1849,6 +1848,63 @@ Steps:
 22. return (x, y)
 ~~~
 
+### Simplified SWU for AB == 0 {#simple-swu-AB0}
+
+Wahby and Boneh {{WB19}} show how to adapt the simplified SWU mapping to
+Weierstrass curves having A == 0 or B == 0, which the mapping of
+{{simple-swu}} does not support.
+(The case A == B == 0 is excluded because y^2 = x^3 is not an elliptic curve.)
+
+This method applies to curves like secp256k1 {{SEC2}} and to pairing-friendly
+curves in the Barreto-Lynn-Scott {{BLS03}}, Barreto-Naehrig {{BN05}}, and other families.
```

Comment:
The reference [KSS08] is not used in the document

Reply:
BLAKE has a wrong reference

Reply:
Oops! Fixed the BLAKE reference, removed KSS08 from the bib (it isn't used on master either; we can bring it back if we need it later).